### PR TITLE
add files for composer support

### PR DESCRIPTION
**Pull request by franklinkim**
Adds composer.json to be able to install your plugin through composer.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "tinymce"
+    ],
+    "description": "Advanced TinyMCE settings",
+    "license": "MIT",
+    "type": "shopware-plugin",
+    "extra": {
+      "installer-name": "SheAdvancedTinyMce"
+    },
+    "require": {
+      "php": "^7.0",
```
Comment (hlohaus) [inline composer.json]: It's require php 7. shopware 5 is php 5.6 too

Comment (franklinkim) [reply composer.json]: set it to 5.6

Comment (franklinkim) [reply composer.json]: sorry, has to be `>=5.6` of cause

Comment (hlohaus) [reply composer.json]: yo have to update the look too

Comment (franklinkim) [reply composer.json]: done

**Comment by franklinkim** [comment]
@hlohaus thx for the merge. Would be great to have tags/releases on github as well so they could be referenced instead of pointing to the master branch

**Pull request merged**